src/components/detail/UserDetail.jsx

```diff
@@ -1,56 +1,189 @@
+    props.user;
+
+  const images = [
+    {
+      label: 'Image',
+      imgPath: {image}, //{image}
+    },
+    {
+      label: 'Imnage with mask',
+      imgPath: {image_with_mask}, //{image_with_mask}
+    },
+    {
+      label: 'Vaccine document',
+      imgPath: {vaccine_document}, //{vaccine_document}
+    },
+  ];
+
+  const theme = useTheme();
+  const [activeStep, setActiveStep] = React.useState(0);
+  const maxSteps = images.length;
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
   return (
-        <Box sx={style}>
-          <Typography gutterBottom variant="h4" component="div"> User Detail </Typography>
-          <Typography gutterBottom variant="h5" component="div">
-            Student <b>{student_name}</b> - student id: <b>{sid}</b>
+    // <ThemeProvider theme={userDetailsTheme}>
+    <Box sx={style}>
+      <Typography variant={'h2'} gutterBottom component="div">
+        User Details
+      </Typography>
+      <Divider variant="middle" />
+      <Grid container spacing={3}>
+        <Grid item xs={6} my={3}>
+          <Typography
+            variant="h4"
+            gutterBottom
+            sx={{ mx: 1, fontWeight: 'lighter' }}
+            component="div"
+          >
+            {student_name} - {sid}
             {STATUS[status]}
           </Typography>
-          <Typography id="modal-modal-title" variant="h6" component="h2">
-            1. User Face Image (Without Mask):
-          </Typography>
-          <img width={'60%'} src={image}/>
-          <Typography id="modal-modal-title" variant="h6" component="h2">
-            2. User Face Image (With Mask):
-          </Typography>
-          <img width={'60%'}  src={image_with_mask}/>
-          <Typography id="modal-modal-title" variant="h6" component="h2">
-            3. Vaccine Document:
-          </Typography>
-          <img width={'60%'}  src={vaccine_document}/>
-          <Typography id="modal-modal-title" variant="h6" component="h2">
-            4. Contact Information:
-          </Typography>
-          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
-            <b> Phone:  </b> {phone}
+
+          <Typography id="modal-modal-description" variant="h5" sx={{ mt: 3, mx: 1 }}>
+            <b> Email: </b> {email}
           </Typography>
-          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
-            <b> Email:  </b> {email}
+          <Typography id="modal-modal-description" variant="h5" sx={{ mt: 2, mx: 1 }}>
+            <b> Phone: </b> {phone}
           </Typography>
-        </Box>
+        </Grid>
+        <Grid item xs={6} my={3}>
+          <Box
+            sx={{
+              maxWidth: 800,
+              flexGrow: 1,
+              bgcolor: '#d5bec4',
+              borderRadius: 2,
+              border: '1px solid', //imageslider bgcolor
```

Comment:
@Minh0811 you should separate the style props into a const that's declared outside of the main component, please refer to the navigation components in the navigation folder for more information.